src/precice/SolverInterface.hpp

```diff
@@ -782,7 +782,9 @@ class SolverInterface {
   /**
    * @brief setBoundingBoxes Define a region of interest on a received mesh
    *        (<use-mesh ... from="otherParticipant />") in order to receive
-   *        only a certain mesh region.
+   *        only a certain mesh region. Have a look at the website under
+   *        [sidebar]->Couple your code -> Advanced topics -> Accessing
+   *        received meshes directly for a comprehensive documentation
```

Comment:
Why not give the url? We will probably not change it in the future.

Reply:
Added..